ByeBoo-iOS/ByeBoo-iOS/Presentation/Feature/WriteQuest/ActivationType/ViewModel/WriteActiveTypeViewModel.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  WriteActiveQuestViewModel.swift
+//  ByeBoo-iOS
+//
+//  Created by 이나연 on 7/13/25.
+//
+
+import Combine
+import Foundation
+
+struct WriteActiveTypeViewModel: ViewModelType {
+    
+    private var cancellables = Set<AnyCancellable>()
+    private(set) var output: Output
+    private let saveQuestTypeUseCase: SaveQuestTypeUseCase
+    private let getQuestInfoUseCase: GetQuestInfoUseCase
+    private let questInfoResultSubject: PassthroughSubject<Result<QuestInfoEntity, ByeBooError>, Never> = .init()
+    private let didSuccessPostSubject: PassthroughSubject<Result<Void, ByeBooError>, Never> = .init()
```

Comment:
저같은 경우 이런 변수들은 종류에 따라서 공백 한 칸씩 주는 걸 선호하긴 합니다.. 
```suggestion
    private var cancellables = Set<AnyCancellable>()
    
    private(set) var output: Output
    
    private let saveQuestTypeUseCase: SaveQuestTypeUseCase
    private let getQuestInfoUseCase: GetQuestInfoUseCase
    
    private let questInfoResultSubject: PassthroughSubject<Result<QuestInfoEntity, ByeBooError>, Never> = .init()
    private let didSuccessPostSubject: PassthroughSubject<Result<Void, ByeBooError>, Never> = .init()
```

Reply:
수정완!!